src/pages/Swap/index.tsx

```diff
@@ -189,6 +202,10 @@ const SwapPageUi: React.FC<SwapPageUiProps> = ({
                   : currentFormValues.toToken,
             }))
           }
+          rightMaxButton={{
+            label: t('swapPage.fromTokenAmountField.max').toUpperCase(),
+            valueOnClick: maxFromInput(),
```

Comment:
Following my previous comment, `maxFromInput` would now be a variable:
```suggestion
            valueOnClick: maxFromInput,
```